content/kb/tutorials/llm/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: LLM Tutorials
+slug: /knowledge-base/tutorials/llm-tutorials
+---
+
+# LLM Tutorial: OpenAI, LangChain, and Streamlit in 18 lines of code
```

Comment:
The naming here feels a bit odd.

I would imagine that eventually we have an LLM tutorial landing page and then a bunch of broken out tutorials (similar to deploy and connections), is that the idea?

To me the most appropriate title for this content in the docs (vs a blog title) would be something like "Getting started with LLM apps" or "LLM app quickstart".

I would vote that we update the title / slug to reflect something like that and have a later PR to add the index page.

Defer to the docs team on this decision, that is just my 2c

Reply:
You've hit it exactly: This is a single tutorial that's going to become a category of tutorials. I was stuck between a generic category "LLM Tutorials" which will look good in navigation (short and clear) and the fact that we are in fact just starting out with one for now. I used the slug of the category it's going to become (plural) but made the visible text for users singular. The title of the tutorial was the more specific version thinking of it as one of many to come.

I'm fine renaming everything as well. "Getting started with LLM apps" feels a little long for an entry in navigation, but "LLM quickstart" is better.